test/util/bitcoin-util-test.py

```diff
@@ -63,87 +67,93 @@ def bctester(testDir, input_basename, buildenv):
-        outs = proc.communicate(input=inputData)
+        outs = proc.communicate(input=input_data)
     except OSError:
-        logging.error("OSError, Failed to execute " + execprog)
+        logging.error("OSError, Failed to execute " + exec_prog)
         raise
 
-    if outputData:
+    if output_data:
         data_mismatch, formatting_mismatch = False, False
         # Parse command output and expected output
         try:
-            a_parsed = parse_output(outs[0], outputType)
+            a_parsed = parse_output(outs[0], output_type)
         except Exception as e:
-            logging.error('Error parsing command output as %s: %s' % (outputType, e))
+            logging.error("Error parsing command output as %s: %s" % (output_type, e))
             raise
         try:
-            b_parsed = parse_output(outputData, outputType)
+            b_parsed = parse_output(output_data, output_type)
         except Exception as e:
-            logging.error('Error parsing expected output %s as %s: %s' % (outputFn, outputType, e))
+            logging.error("Error parsing expected output %s as %s: %s" % (output_fn, output_type, e))
             raise
         # Compare data
         if a_parsed != b_parsed:
-            logging.error("Output data mismatch for " + outputFn + " (format " + outputType + ")")
+            logging.error("Output data mismatch for " + output_fn + " (format " + output_type + ")")
             data_mismatch = True
         # Compare formatting
-        if outs[0] != outputData:
-            error_message = "Output formatting mismatch for " + outputFn + ":\n"
-            error_message += "".join(difflib.context_diff(outputData.splitlines(True),
+        if outs[0] != output_data:
+            error_message = "Output formatting mismatch for " + output_fn + ":\n"
+            error_message += "".join(difflib.context_diff(output_data.splitlines(True),
                                                           outs[0].splitlines(True),
-                                                          fromfile=outputFn,
+                                                          fromfile=output_fn,
                                                           tofile="returned"))
             logging.error(error_message)
             formatting_mismatch = True
 
         assert not data_mismatch and not formatting_mismatch
 
     # Compare the return code to the expected return code
-    wantRC = 0
-    if "return_code" in testObj:
-        wantRC = testObj['return_code']
-    if proc.returncode != wantRC:
-        logging.error("Return code mismatch for " + outputFn)
+    want_rc = 0
+    if "return_code" in test_obj:
+        want_rc = test_obj["return_code"]
+    if proc.returncode != want_rc:
+        logging.error("Return code mismatch for " + output_fn)
         raise Exception
 
-    if "error_txt" in testObj:
-        want_error = testObj["error_txt"]
+    if "error_txt" in test_obj:
+        want_error = test_obj["error_txt"]
```

Comment:
I don't think it makes sense to replace every variable in the whole file by a lower_case name.